pkgs/unified_analytics/lib/src/event.dart

```diff
@@ -266,13 +266,27 @@ final class Event {
           if (error != null) 'error': error,
         };
 
+  /// Event that is emitted from Flutter returning the results of a pub run.
+  ///
+  /// [context] - the context with which pub was run such as "create" and "get".
+  ///
+  /// [result] - string value that is either "success" or "failure".
+  Event.flutterPubResult({
```

Comment:
in the future, it would help reviewing these if they you linked a corresponding tool change, so I can see how the event will be called, and what from package:usage it's replacing.

Is this meaning to replace: https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/commands/packages.dart#L355

And if so, do we not need the timing?

Reply:
That's a good idea, i'll incorporate that in the future PRs but there are actually only 2 call sites for this event, an example is [here](https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/dart/pub.dart#L435-L439)

The timing events will be handled separately, I am thinking through if it makes sense to have just one timing event for all dash tools or each tool having its own

Reply:
ahh, ok, thanks.